### PR TITLE
readme file and readmeObjects

### LINKED ISSUE #14: Add README.md File
you should create readme.md file to describe your work in the repository

### PR DESCRIPTION
**Pull request by oshriFadlon1**
I add readme file and all the object that need for it.
closes #14

### CODE REVIEW

**Review by Danielsio — APPROVED** [review]

**Comment by Danielsio** [comment]
can you do PR of readme for Plan-IT-vite as well ?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+![Logo](https://github.com/oshriFadlon1/Plan_It_Enigne_1.0/blob/master/Plan.png)
```
Comment (itayf9) [inline README.md]: I think this link has the wrong url

**Review by itayf9 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+The calendar creation screen and generate it 
+![App Screenshot](readme_objects/generate.png)
+
+## The project programmers
+
+- [@Danielsio](https://github.com/Danielsio)
+
+- [@itayf9](https://github.com/itayf9)
+
+- [@oshriFadlon1](https://github.com/oshriFadlon1)
```
Comment (Danielsio) [inline README.md]: drop the @

**Review by Danielsio — APPROVED** [review]

**Review by itayf9 — APPROVED** [review]
Seems good to me

**Pull request merged**